perf-sq.py

```diff
@@ -0,0 +1,8 @@
+#check if a number is a perfect square in raneg 0-100
+n=int(input('enter the number to be checked'))
+for i in range(1,100):

```

Comment:
Could we turn this into a while loop and perhaps have a higher limit. 100 seems a bit low.